scripts/entities_csv2json.py

```diff
@@ -87,10 +87,11 @@
                     line = line.strip()
                     if line:
                         rawSynonyms = line.split(';')
-                        # strip and lower all items in line
-                        [x.strip().lower() for x in rawSynonyms]
-                        representativeValue = rawSynonyms[0]
-                        synonyms = sorted(list(set(rawSynonyms[1:])))
+                        representativeValue = rawSynonyms[0].strip() # keep the value as it is to match in conditions
+                        rawSynonyms = [x for x in [x.strip().lower() for x in rawSynonyms] if len(x) > 0] # strip and lower all items in line
+                        synonyms = set(rawSynonyms) # remove duplicities
+                        synonyms.remove(representativeValue.lower()) # remove value from synonyms, so that duplicity with value is not possible
+                        synonyms = sorted(list(synonyms))
```

Comment:
strip() and lower() took no effect without assignment to rawSynonyms. There could be diplicate values in synonyms.
Added removal of empty strings.

Reply:
Merged with current state.